### ISSUE TITLE
[Data][Docs] Simplify HuggingFace quick start code

### ISSUE THREAD

**Issue by bveeramani** [ray-project/ray#35342]
### Description
**This issue is blocked on https://github.com/ray-project/ray/issues/35340**

To simplify the quick start code, we shouldn't unwrap the model output.

```diff
class HuggingFacePredictor:
    ...
    def __call__(self, batch: Dict[str, np.ndarray]):
        model_out = self.model(list(batch["data"]), max_length=20, num_return_sequences=1)
-      batch["output"] = [sequence[0]["generated_text"] for sequence in model_out]
+      batch["output"] = model_out
        return batch
```

This issue is blocked on #[data] 

### Link

https://docs.ray.io/en/master/data/batch_inference.html#quick-start

**Comment by bveeramani** [comment]
cc @pcmoritz @ericl

**Comment by bveeramani** [comment]
Closed by #35688
